assets/main-menu-dropdown.js

```diff
@@ -1,9 +1,11 @@
 $("[data-toggle='ecosystem-dropdown']").on("click", function() {
   toggleDropdown($(this).attr("data-toggle"));
+  toggleArrowClass(".ecosystem-option");
 });
 
 $("[data-toggle='resources-dropdown']").on("click", function() {
   toggleDropdown($(this).attr("data-toggle"));
+  toggleArrowClass(".resource-option");
 });
```

Comment:
When I said that these two click functions are pretty similar, I meant that you can refactor and pass both data-toggle selector arguments to a single function. Then inside of the single function, you can toggle the dropdown and arrow for both menus. With this approach, you wouldn't need to create a separate function for toggling the arrow. It's not super necessary, but since we're adding two lines of code to two functions that do the same thing, it makes sense to streamline it a bit.